src/tests/nativeaot/SmokeTests/UnitTests/UnitTests.csproj

```diff
@@ -8,6 +8,9 @@
     <!-- There's just too many of these warnings -->
     <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
     <NoWarn>$(NoWarn);IL3050;IL3054</NoWarn>
+    <DefineConstants Condition="'$(TargetsiOS)' == 'true'">$(DefineConstants);TARGET_IOS</DefineConstants>
```

Comment:
The tests are build and implemented as platform neutral. It would be better to replace this with `if (!OperatingSystem.Is...() || ...)` in the test sources.